<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleTodoEditor</class>
 <widget class="QDialog" name="SimpleTodoEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑待办事项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>标题:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSubject"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>完成度:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="rdoUnfinished">
         <property name="text">
          <string>未开始</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdoProcessing">
         <property name="text">
          <string>进行中</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdoFinished">
         <property name="text">
          <string>已完成</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimpleTodoEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimpleTodoEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
